docs/userguides/cache.md

```diff
@@ -0,0 +1,69 @@
+# Cache
+
+Use the cache plugin to store provider data in an sqlite database.
+
+```bash
+ape cache init --network <ecosystem-name>:<network-name>
+```
+
+If you want to set up your network connection for caching use [this guide](./network.html)
+
+```bash
+ape cache init --network ethereum:mainnet
+```
+
+This creates a SQLite database file in the hidden ape folder.
+
+## Get data from the provider
+
+Use `ape console`:
+
+```bash
+ape console --network ethereum:mainnet:infura
+```
+
+Run a few queries:
+
+```python
+In [1]: chain.blocks.query("*", stop_block=20)
+In [2]: chain.blocks[-2].transactions
+```
+
+On a deployed contract, you can query events:
+- Below, FooHappened is the event from your contract instance that you want to query from.
+
+```python
+contract_instance.FooHappened.query("*", start_block=-1)
+```
+
+where `contract_instance` is the return of owner.deploy(Contract)
+
+See [this guide](../userguides/contracts.html) for more information how to get a contract instance.
+
+Exit the IPython interpreter.
+
+You can query the cache database directly, for debugging purposes. For example, to get the `blocks` table data from the SQLite db we can do the following:
+```bash
+ape cache query --network ethereum:mainnet:infura "SELECT * FROM blocks"
```

Comment:
I follow this guide and everything works until I get to this part.
I am seeing the DBs get created and I do the console step to query blocks from the provider and I see the results show up when I do that.

But then when I get to this part, the CLI does not output anything.


```
ape cache query --network starknet:testnet "SELECT * FROM blocks
ape cache query --network ethereum:mainnet:infura "SELECT * FROM blocks
ape cache query --network ethereum:mainnet:infura "SELECT * FROM blocks"
```

Reply:
I don't have an answer on why that isn't working for you at the moment, I will review

Reply:
I think if it doesn't find any data, it will not output anything

Reply:
Yes, that is the functionality. You said you went through the markdown, but just to confirm, you did run a query through the console before you tried to run the query via `ape cache query` correct?

Reply:
Yes, I run the blocks query from the markdown guide in each of the ecosystem-network combos I tried